lib/routes/constraints/translations.rb

```diff
@@ -6,21 +6,23 @@ class Translations
       def matches?(request)
         @params = request.params
 
-        if show_route?
+        if member_route?
            valid_show_id? && has_translated_type_param?
```

Comment:
Does `valid_show_id?` need to be renamed to `valid_member_id?`?

Reply:
yeah - it probably should for consistent naming. I can update.